pkg/v7/chartconfig/chartconfig.go

```diff
@@ -0,0 +1,69 @@
+package chartconfig
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/tenantcluster"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Config represents the configuration used to create a new chartconfig service.
+type Config struct {
+	Logger micrologger.Logger
+	Tenant tenantcluster.Interface
+
+	ProjectName string
+}
+
+// Service provides shared functionality for managing chartconfigs.
+type Service struct {
```

Comment:
I would call it `ChartConfig`. We had the issue already that just everything was called service in certain projects. The naming would be more explicit somehow.